UniversalAlgebra/Structures/Graphs/Basic.lagda

```diff
@@ -15,6 +15,8 @@ The *graph* of 𝑨 is the structure Gr 𝑨 with the same domain as 𝑨 with r
 
 {-# OPTIONS --without-K --exact-split --safe #-}
 
+open import Agda.Primitive using    ( Level )
```

Comment:
It's only useful to do this if the module itself is parametrized by a level - otherwise it is preferred to have the imports be inside the module itself.

On the other hand, it is indeed better to minimize the use of `Agda.*` modules. You really only need one, as you want to rename `Set` to `Type` - the rest should grabbed from the library.

Reply:
Yes, that import was left over from when I was experimenting with adding some level parameters to the module.  When I decided against that I forgot to remove the import... but thanks for noticing it.

(fixed in next PR update)

Reply:
fixed